flixel/addons/editors/tiled/TiledTileSet.hx

```diff
@@ -92,23 +74,125 @@ class TiledTileSet
 			{
 				if (!node.has.id)
 				{
-					continue;
+					tileHeight = Std.parseInt(source.att.tileheight);
+				}
+				if (source.has.spacing) 
+				{
+					spacing = Std.parseInt(source.att.spacing);
+				}
+				if (source.has.margin) 
+				{
+					margin = Std.parseInt(source.att.margin);
 				}
 				
-				var id:Int = Std.parseInt(node.att.id);
-				tileProps[id] = new TiledPropertySet();
-				tileProps[id].keys.set("id", Std.string(id));
-				for (prop in node.nodes.properties)
+				// read properties
+				properties = new TiledPropertySet();
+				for (prop in source.nodes.properties) {
+					properties.extend(prop);
+				}
+				
+				// read tiles properties
+				tileProps = new Array<TiledPropertySet>();
+				
+				for (node in source.nodes.tile)
+				{
+					if (!node.has.id)
+					{
+						continue;
+					}
+					
+					var id:Int = Std.parseInt(node.att.id);
+					tileProps[id] = new TiledPropertySet();
+					tileProps[id].keys.set("id", Std.string(id));
+					for (prop in node.nodes.properties)
+					{
+						tileProps[id].extend(prop);
+					}
+				}
+				
+				// read tiles images
+				tileImagesSources = new Array<TiledImageSet>();
+				
+				for (node in source.nodes.tile)
 				{
-					tileProps[id].extend(prop);
+					if (!node.has.id)
+					{
+						continue;
+					}
+					
+					var id:Int = Std.parseInt(node.att.id);
+					tileImagesSources[id] = new TiledImageSet();
+					tileImagesSources[id].keys.set("id", Std.string(id));
+					tileImagesSources[id].extend(node);
 				}
+				
+				if (tileWidth > 0 && tileHeight > 0)
+				{
+					numRows = 0;//cast(imgWidth / tileWidth);
```

Comment:
Sorry, I think something didn't uploaded properly. I though I changed this. I have trouble getting used to Github. 

Now I'm looking that I have the changes locally but they didn't uploaded.

Same for next comment.